include/rocksdb/table.h

```diff
@@ -490,6 +490,25 @@ struct BlockBasedTableOptions {
 
   PrepopulateBlockCache prepopulate_block_cache =
       PrepopulateBlockCache::kDisable;
+
+  // If true, an estimated memory usage of table building
+  // will be charged to the block cache as memory accounting
+  // if block cache available .
+  //
+  // Charged memory usage includes:
+  // 1. (new) Bloom Filter and Ribbon Filter construction
+  // 2. More to come...
+  //
+  // Note:
+  // 1. (new) Bloom Filter and Ribbon Filter construction
+  //
+  // If banding portion of Ribbon Filter uses up too much memory
+  // relative to the avaible space left in the block cache
+  // at that point (i.e, causing a cache full when strict_capacity_limit =
+  // true), construction will fall back to (new) Bloom Filter.
+  //
+  // Default: false
```

Comment:
Todo:
Move the option closer to block cache related option and explicit state that block cache needs to be available for this option to take effect

Reply:
Done